BridgeAppSDK/SBAProfileManager.swift

```diff
@@ -143,6 +149,9 @@ open class SBAProfileManager: SBADataObject, SBAProfileManagerProtocol {
         return allItems
     }()
     
+    private dynamic var demographicSchemaIdentifier: String?
+    private dynamic var demographicArchiveFilename: String?
```

Comment:
I'm not seeing where these are added to the list of keys to parse out of the dictionary. How are they set?